src/HttpClientInterception/HttpRequestInterceptionBuilder.cs

```diff
@@ -97,6 +97,44 @@ public HttpRequestInterceptionBuilder For(Predicate<HttpRequestMessage>? predica
         return this;
     }
 
+    /// <summary>
+    /// Configures the builder to match any request that meets the criteria defined by the specified predicate.
+    /// </summary>
+    /// <param name="predicates">
+    /// Two or more delegates to a method which returns <see langword="true"/> if the
```

Comment:
The "Two or more" bit is a bit redundant here too, as the behaviour is essentially the same for > 0 delegates.